packages/bootstrap-vue-next/src/components/BButton/BButton.vue

```diff
@@ -6,21 +6,12 @@
     v-bind="computedAttrs"
     @click="clicked"
   >
-    <div
-      v-if="loadingBoolean"
-      class="btn-loading"
-      :class="{'mode-fill': loadingMode === 'fill', 'mode-inline': loadingMode === 'inline'}"
-    >
-      <slot name="loading">
-        <BSpinner class="btn-spinner" :small="size !== 'lg'" />
-      </slot>
-    </div>
-    <div
-      class="btn-content"
-      :class="{'btn-loading-fill': loadingBoolean && loadingMode === 'fill'}"
-    >
+    <slot v-if="loadingBoolean" name="loading">
+      <BSpinner class="btn-spinner" :small="size !== 'lg'" />
+    </slot>
+    <span v-if="(loadingBoolean && !fillBoolean) || !loadingBoolean" class="btn-content">
```

Comment:
If the classes can be removed the span here could just use a template fragment.
```suggestion
    <template v-if="(loadingBoolean && !fillBoolean) || !loadingBoolean">
```
Though, I don't think it makes much of a difference.

Reply:
> If the classes can be removed the span here could just use a template fragment.
> 
> Though, I don't think it makes much of a difference.

We can work with removing .btn-content.
But I'm not quite sure with removing .btn-spinner

Reply:
nvm my bad, changes coming